front/app/containers/UsersShowPage/Submissions/index.tsx

```diff
@@ -7,7 +7,7 @@ import { ideaDefaultSortMethodFallback } from 'api/phases/utils';
 import useUserIdeasCount from 'api/user_ideas_count/useUserIdeasCount';
 import useUserBySlug from 'api/users/useUserBySlug';
 
-import { IdeaCardsWithoutFiltersSidebar } from 'components/IdeaCards';
+import { IdeaCardsWithFiltersSidebar } from 'components/IdeaCards';
```

Comment:
I'm not sure about this. Does it make sense to have filters on a user's page?

Reply:
I think you're right - I'll remove it :)